### PR TITLE
feat: static homapage

### PR DESCRIPTION
**Pull request by vikrantkode**
### Static Homepage Layout with Auth

image

### CODE REVIEW

**Review by harshitpaliwal95 — APPROVED** [review]
nothing much to review in code
- Ui looks clean 
- good file folder structure

File: src/Components/CreatePost.jsx [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import React from 'react'
+
+const CreatePost = () => {
+  return (
+    <div className="border-2 rounded-md p-4">
+        <textarea placeholder='Write Something Intresting...' className='border-none focus:outline-none w-full'/>
```
Comment (sashaboi) [inline src/Components/CreatePost.jsx]: resize: none;  would help not break changes

File: src/Components/PostCard.jsx [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+        <div className="flex">
+         <img src="https://res.cloudinary.com/mriant2812/image/upload/v1653508856/cld-sample.jpg" alt="profile_pic" className='h-8 w-8 rounded-full cursor-pointer self-center'/>
+         <h1 className='font-semibold ml-2'>Sanskaari baalak</h1>
+         <p className='ml-4 text-gray-400'>@sanskaari_baalak</p>
+        </div>
+        <div className="mt-4 mb-4">
+            <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Labore, nesciunt esse expedita, provident officia dolore quaerat quisquam, architecto ipsam eius voluptas quis mollitia iusto? Fuga!</p>
+        </div>
+        <div className="flex items-center justify-between gap-4">
+         <label className="flex items-center"><FaRegHeart/>  1 Like</label>
```
Comment (sashaboi) [inline src/Components/PostCard.jsx]: on hover property cursor pointer might help ux for all action buttons

File: src/Components/Search.jsx [hunk 3]
```diff
@@ -0,0 +1,14 @@
+import React from 'react'
+import { FcSearch } from "react-icons/fc";
+const Search = () => {
+  return (
+    <div className="flex items-center mt-5 w-full md:w-full h-10">
+        <FcSearch className='absolute top-22 left-19 z-10 mr-2'/>
```
Comment (sashaboi) [inline src/Components/Search.jsx]: increase the negative margin for the search icon to make it look like it is no on the edge of the bar

**Review by sashaboi — APPROVED** [review]
looks good to merge now, looking forward to functionality of the app
sidebars could have a little shadow to make then stand out
Do handle other screen size not only smaller but also bigger like 1920*1080 and 4k resolutions. use vh and hw for any issues

**Comment by vikrantkode** [comment]
Thanks for review

**Pull request merged**